cvem/connectors/one/OpenNebula.py

```diff
@@ -148,10 +148,10 @@ def get_vm_list():
 			res = []
 			for vm in res_vm.VM:
 				host = HostInfo(int(vm.HISTORY_RECORDS.HISTORY[0].HID), vm.HISTORY_RECORDS.HISTORY[0].HOSTNAME)
-				new_vm = VirtualMachineInfo(int(vm.ID), host, vm.TEMPLATE.MEMORY * 1024, vm)
+				new_vm = VirtualMachineInfo(int(vm.ID), host, int(vm.TEMPLATE.MEMORY) * 1024, vm)
 				new_vm.user_id = vm.UID
 				if vm.USER_TEMPLATE.MEM_TOTAL:
-					new_vm.set_memory_values(int(vm.USER_TEMPLATE.MEM_TOTAL_REAL),
+					new_vm.set_memory_values(int(vm.REALMEMORY),
```

Comment:
The real_memory is the amount of memory assigned to the VM (in most of the cases is the same as allocated_memory). Here you are using REALMEMORY defined as the current memory usage.